packages/frontend/src/components/Header/SearchDropdown.tsx

```diff
@@ -0,0 +1,104 @@
+import cx from 'classnames';
+import { useTranslation } from 'react-i18next';
+import styles from './search.module.css';
+import { ChevronRightIcon } from '@navikt/aksel-icons';
+import { autoFormatRelativeTime, useSavedSearches } from '../../pages/SavedSearches';
+import { SavedSearchesFieldsFragment } from 'bff-types-generated';
+import { getPredefinedRange } from '../FilterBar/dateInfo';
+import savedSearchStyles from '../../pages/SavedSearches/savedSearches.module.css';
```

Comment:
hvorfor gjenbrukes styles for saveSearches her? burde ikke det ligge i `search.module.css`, eller enda bedre, `searchDropdown.module.css` for at disse komponentene er litt mer frikoplet?

Reply:
Hadde jeg kopiert syles hadde jeg garantert fått kommentar på at det ikke er DRY nok. Men kan godt legge de inn begge steder.